### ISSUE TITLE
Dataset processing issues

### ISSUE THREAD

**Issue by xiojioji** [ArnabKumarRoy02/ResEmoteNet#26]
Hey, bro, I have seen the datasets you processed, but I found that the training set here has nearly 3,000 more pictures than the original RAF-DB training set, and the validation set also has about 6,000 pictures. Can I ask how you processed these extra data?
